src/components/__tests__/search-bar-test.jsx

```diff
@@ -16,20 +16,40 @@ describe('search-bar', () => {
   });
 
   it('Handles search queries with a protocol correctly', () => {
-    const searchBar = shallow(<SearchBar onSearch={onSearch} />);
+    const onSearch = jest.fn();
+    const searchBar = shallow(<SearchBar onSearch={onSearch} startDate={null} endDate={null} />);
     const searchBarInput = searchBar.find('input');
     searchBarInput.simulate('change', {target: {value: 'http://epa'}});
 
     jest.runAllTimers();
-    expect(onSearch).toHaveBeenCalledWith({url: 'http://epa*'});
+    expect(onSearch).toHaveBeenCalledWith('http://epa*', null, null );
   });
 
   it('Handles search queries without a protocol correctly', () => {
-    const searchBar = shallow(<SearchBar onSearch={onSearch} />);
+    const onSearch = jest.fn();
+    const searchBar = shallow(<SearchBar onSearch={onSearch} startDate={null} endDate={null} />);
     const searchBarInput = searchBar.find('input');
     searchBarInput.simulate('change', {target: {value: 'epa'}});
 
     jest.runAllTimers();
-    expect(onSearch).lastCalledWith({url: '*//epa*'});
+    expect(onSearch).toHaveBeenCalledWith('*//epa*', null, null );
+  });
+
+  it('Handles date range search queries for startDate', () => {
+    const onSearch = jest.fn();
+    const searchBar = mount(<SearchBar inputId="1" onSearch={onSearch} startDate={null} endDate={null} />);
+    searchBar.find('input#startDate1').simulate('focus');
+
+    searchBar.find('.CalendarDay.CalendarDay_1').first().simulate('click');
+    expect(onSearch.mock.calls[0][1]).toBeInstanceOf(moment);
```

Comment:
If you want to be a bit more fancy, Jest gives you a way to check types when matching call arguments or objects:

```suggestion
    expect(onSearch).toHaveBeenCalledWith(null, expect.any(moment), null);
```

**But that’s only very slightly clearer than what you already have, so ¯\\\_(ツ)\_/¯**

(Same for the other test below.)